src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -55,7 +55,7 @@ class CallActor(call: Call, locallyQualifiedInputs: CallInputs, backend: Backend
   when(CallNotStarted) {
     case Event(Start, _) =>
       val backendInputs = backend.adjustInputPaths(call, locallyQualifiedInputs)
-      call.instantiateCommandLine(backendInputs) match {
+      call.instantiateCommandLine(backendInputs, backend.engineFunctions(backend.setupCallEnvironment(call, workflowDescriptor))) match {
```

Comment:
`TaskExecutionContext` (return value of backend.setupCallEnvironment) is supposed to represent the the thing you mentioned.  We could move `.engineFunctions` as a method off of `TaskExecutionContext`